src/VerifyEmailHelperInterface.php

```diff
@@ -33,6 +36,8 @@ interface VerifyEmailHelperInterface
     public function generateSignature(string $routeName, string $userId, string $userEmail, array $extraParams = []): VerifyEmailSignatureComponents;
 
     /**
+     * @deprecated since v1.17.0, use validateEmailConfirmationFromRequest instead.
```

Comment:
Is this even the right way to deprecate an interface method?

Reply:
To the best of my knowledge, yes - but not sure.